docs/resources/samples/email-chart-image.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Email chart image'
+description: 'Learn how to use Office Scripts and Power Automate to extract and email chart image.'
+ms.date: 03/04/2021
+localization_priority: Normal
+---
+
+# Use Office Scripts and Power Automate to email chart and table images
+
+Shows how to use Office Scripts and Power Automate actions to recalculate, create a chart, and extract images as base-64 strings to be used later to send by email.
+
+## Example scenario
+
+* Calculate to get latest results
+* Create chart
+* Get chart and table images
+* Email the images to recipient using Power Automate action
+
+## Screenshots
+
+### Input data
+
+![Input data](../../images/input-data.png)
+
+### Output chart
+
+![Chart created](../../images/chart-created.png)
+
+## Email that was received through Power Automate flow
+
+![Email received](../../images/email-received.png)
+
+## Solution
+
+The solution has 2 parts:
+
+1. [Office Script to calculate and extract Excel chart and table](#office-script)
+1. A Power Automate flow to invoke script and email the results.
```

Comment:
Where's the PA flow? I don't think we can really publish this sample without it.

Reply:
Tracking with [feature](https://office.visualstudio.com/OC/_workitems/edit/4924924)